### DISCUSSION TITLE
delete project in vercel

### DISCUSSION THREAD

**Discussion by AngelC-dev** [vercel/community#2356] [Help] [answered]
### Summary

in project settings delete, enter project name: angelc-dev/angel-c-blog, verify: angelc-dev/angel-     , will not let me complete to verify, cannot delete project. any suggestions

### Additional information

please match the requested format,
cannot complete to verify

**Comment by amyegan** [comment, marked as answer]
Hi @AngelC-dev! The second box is asking that you type "delete my project" to verify you are ready to delete it.

![image](https://user-images.githubusercontent.com/8247105/236840882-2c0b5760-6574-445d-bfbd-b4ec6d5c0291.png)

**Reply by AngelC-dev** [reply]
haha, you where right, I wasn't reading / was too tired to read complete instruction, thanks. btw, do you know any nextjs, having issues build / exporting static generate app code?? in case you have had same issue

**Reply by amyegan** [reply]
No worries! It's intentionally a little tricky so people don't accidentally delete the wrong project. 

I know quite a bit about working with Next.js, but the real experts on that are over in [Next.js Discussions](https://github.com/vercel/next.js/discussions). So feel free to ask in a new discussion post here or over there for whatever advice you need 😄

**Reply by kingcwt** [reply]
乖乖
